modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/resource/v1_0/RelatedObjectEntryResourceImpl.java

```diff
@@ -283,21 +284,15 @@ private ObjectEntry _getRelatedObjectEntry(
 			relatedObjectDefinition, relatedObjectEntryId);
 	}
 
-	private ObjectDefinition _getSystemObjectDefinition(String previousPath) {
+	private ObjectDefinition _getSystemObjectDefinition(String previousPath)
+		throws Exception {
+
 		SystemObjectDefinitionMetadata systemObjectDefinitionMetadata =
 			_getSystemObjectDefinitionMetadata(previousPath);
 
-		ObjectDefinition systemObjectDefinition =
-			_objectDefinitionLocalService.fetchSystemObjectDefinition(
-				systemObjectDefinitionMetadata.getName());
-
-		if (systemObjectDefinition != null) {
-			return systemObjectDefinition;
-		}
-
-		throw new NotFoundException(
-			"No system object definition metadata for name \"" +
-				systemObjectDefinitionMetadata.getName() + "\"");
+		return _objectDefinitionLocalService.getSystemObjectDefinition(
```

Comment:
Just a question, as I fixed some time ago a bug with deactivated ObjectDefinitions; in this case it is working or we should check that too?

Reply:
I am not aware of what bug did you fix.
Should be a test to check that, right?
Any way, I am just changing the way we get the system object definitions, that should not be allow to be disabled 🤷 

If your question is if I checked something deactivating ObjectDefinition, the answer is no 😅

Reply:
Ok, don't worry, we could test it in the future